lib/broccoli/fastboot-config.js

```diff
@@ -98,13 +98,16 @@ FastBootConfig.prototype.buildDependencies = function() {
 };
 
 FastBootConfig.prototype.readAssetManifest = function() {
+  if (!this.assetMapEnabled) return '';
```

Comment:
Since we know this before the try/catch block I decided to leverage it to do an early exit. Let me know if you'd rather I didn't.